src/store/Controller.h

```diff
@@ -58,7 +58,8 @@ class Controller: public Storage
     /// Warning: The returned StoreEntry is not synced and may be marked for
     /// deletion. Use it only for extracting transaction callback details.
     /// TODO: Group and return just that callback-related data instead?
-    StoreEntry *findCallback(const cache_key *);
+    /// XXX: do not use: may return private entries from store_table
```

Comment:
To avoid giving a negative "do not use" instruction that directly contradicts the instructions given two lines earlier, let's explain what is actually going on and offer a (more) constructive advice:

```
/// Warning: The returned StoreEntry is not synced and may be marked for
/// deletion. It can only be used for extracting transaction callback details.
/// New code should be designed to avoid this deprecated API.
StoreEntry *findCallbackXXX(const cache_key *);
```

Reply:
Done.